subprojects/language-java/src/test/groovy/org/gradle/external/javadoc/internal/MultilineMultiValueJavadocOptionFileOptionTest.groovy

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.external.javadoc.internal
+
+import com.google.common.collect.Lists
+import spock.lang.Specification
+
+class MultilineMultiValueJavadocOptionFileOptionTest extends Specification {
```

Comment:
Added test cases which were missed for `MultilineMultiValueJavadocOptionFileOption`